app/src/main/java/me/saket/dank/ui/webview/ChromeCustomTabsForwardingActivity.java

```diff
@@ -40,8 +42,11 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
     CustomTabsIntent customTabsIntent = builder.build();
 
     String url = getIntent().getStringExtra(KEY_URL);
-    customTabsIntent.launchUrl(this, Uri.parse(url));
-
-    finish();
+    try {
+      customTabsIntent.launchUrl(this, Uri.parse(url));
+      finish();
+    } catch (ActivityNotFoundException exception) {
+      Toast.makeText(this, "Unable to find a browser :(", Toast.LENGTH_SHORT).show();
```

Comment:
I can see that the app already has a string which could fit here `common_error_no_app_found_to_handle_url`. Please use it instead of hard coded value.